.vscode/settings.json

```diff
@@ -2,5 +2,8 @@
     "dotenv.enableAutocloaking": false,
     "workbench.colorCustomizations": {
       "statusBar.background": "#007acc"
-    }
+    },
+    "githubPullRequests.ignoredPullRequestBranches": [
+      "develop"
+    ]
```

Comment:
intentional?

Reply:
yeah, I forgot to mention after the PR was sent

I keep getting notifications from VSCode about a PR on `develop` (I think it doesn't expect to point to a branch other than `main`/`master` that have PRs pointing to it?).

This should stop showing those notifications.